### ISSUE TITLE
Flashing without killing TPM?

### ISSUE THREAD

**Issue by janipewter** [n4ru/1vyrain#41]
I have three T430's all with i5-3320M and 8GB RAM and I am very interested in this mod purely to remove the WiFi white list as I want to put 11ac cards in all of them. Two of the laptops are Windows 10 with BitLocker enabled which I need to have for corporate reasons.

My question - is it safe to do this BIOS mod and not kill the TPM? I can live with a warning at boot, as long as it doesn't require user intervention such as pressing a key to continue etc.

I did see in the long FAQ about taking ownership of the TPM but I am not really sure what this means or how to do it.

Thanks

**Comment by digmorepaka** [comment]
There should be an option inside the security chip configuration for this, or in the utility for TPM in your os

**Comment by janipewter** [comment]
Thanks for the reply, so you are saying that if I manage to take ownership of the TPM before I do the flash, the TPM will still work properly afterwards?

**Comment by digmorepaka** [comment]
Yes

**Comment by janipewter** [comment]
Update: I tried this and the TPM does not work once the modified BIOS is flashed. There is no option in BIOS to clear the TPM and trying to do it through Windows also doesn't work. BitLocker will not enable.

I rolled back with IVprep and then tried clearing the TPM in Windows, which worked this time, and now the TPM functioned again and BitLocker would enable (I didn't go through with it). So I tried flashing with 1vyrain again and once again the TPM stopped functioning. Rolled back, and cleared TPM in Windows, and it worked. Really not sure what else to try now.
